lib/pptx.js

```diff
@@ -3,4 +3,9 @@ let { Slide } = require('./slide');
 module.exports = {
     Presentation: require('./presentation'),
     Slide: Slide,
+    ShapeTypes: require('./shape-types').ShapeTypes, // TODO: I don't like this here...
 };
+
+// Some aliases I like for shape types (Greg D.):
+module.exports.ShapeTypes.TRIANGLE = module.exports.ShapeTypes.ISOSCELES_TRIANGLE;
```

Comment:
Can't you define these aliases inside the ShapeType class itself?

Reply:
Originally I had. Then I didn't like the idea of modifying officegen's original source file.

Reply:
it's our source file now :-)